Sources/Event Logging (Swift)/TrackEventPersistenceService.swift

```diff
@@ -0,0 +1,81 @@
+#if SWIFT_PACKAGE
+import AutomatticTracksModel
+import AutomatticTracksModelObjC
+#endif
+import CoreData
+
+/// Should eventually replace objC class `TracksEventPersistenceService`.  Unfortunately
+/// Swift Packages do not support mixed source classes so we'll instead need to compose.
+///
+@objc
+public class TrackEventPersistenceService: NSObject {
```

Comment:
Is it intentional that this is called **Track**EventPersistenceService, instead of **Tracks**? That does give it a distinct name from the ObjC one, but it seems like it would be really easy to confuse the two. I might recommend something like `TracksEventPesistenceServiceSwift` or something instead.

Reply:
Yes and no.  I had tried using a different name to keep the swift file in the original package.

I'll resort to the idea of keeping the same name and having `Swift` in the end.